lib/pdf_fill/filler.rb

```diff
@@ -6,14 +6,14 @@ module PdfFill
   module Filler
     module_function
 
-    PDF_FORMS = PdfForms.new('pdftk')
+    PDF_FORMS = PdfForms.new(Settings.binaries.pdftk)
     FORM_CLASSES = {
       '21P-527EZ' => PdfFill::Forms::VA21P527EZ
     }.freeze
 
     def combine_extras(old_file_path, extras_generator)
       if extras_generator.text?
-        file_path = "tmp/pdfs/form_#{Time.zone.now}_final.pdf"
+        file_path = "#{old_file_path.gsub('.pdf', '')}_final.pdf"
```

Comment:
To @kreek's question about PII above, there could be a problem with it in the filename here right? Does this also introduce a slight chance that filenames could collide if two people use the same name?